frontend/public/components/build-config.tsx

```diff
@@ -288,13 +289,17 @@ export const BuildConfigsList: React.SFC<BuildConfigsListProps> = (props) => {
     [builds, buildsLoaded, buildsLoadError],
   );
 
-  const buildResource = props.data.map((buildConfig) => {
-    buildConfig.latestBuild =
-      data.builds.latestByBuildName[
-        `${buildConfig.metadata.name}-${buildConfig.metadata.namespace}`
-      ];
-    return buildConfig;
-  });
+  const buildResource = _.isUndefined(props.data)
+    ? []
+    : props.data.map((buildConfig) => {
+        if (props.data) {
```

Comment:
This should not be necessary since we are already checking if the props.data undefined and also map function is handling empty array

Reply:
Tried to reformat the code as suggested, removed the unnecessary lodash import and use of its function `isUndefined`.